src/components/HomePage/FAQSection.js

```diff
@@ -113,44 +113,54 @@ function FAQSection() {
 	const faqs = [
 		{
 			question: `What’s a hackathon?`,
-			answer: `A hackathon is an event where individuals get together for a
-				short period of time to work on a project. These events usually last 24
-				or 36 hours and take place during the weekend. Hack on the Hill is
-				designed to simulate a real hackathon over a 12-hour period.`
+			answer:
+				<>
+					A hackathon is an event during which people from different backgrounds
+					come together to create a project from start to finish in a limited time frame.
+					Check out some examples <Link href='https://hoth7.devpost.com/project-gallery'>here</Link>!
```

Comment:
Don't use Gatsby's Link component for [external links](https://www.gatsbyjs.com/docs/linking-between-pages/#using-a-for-external-links). Instead, use `<a href="…">`.